ci-operator/step-registry/aws/provision/iam-user/minimal-permission/aws-provision-iam-user-minimal-permission-ref.yaml

```diff
@@ -1,15 +1,18 @@
 ref:
   as: aws-provision-iam-user-minimal-permission
-  from_image:
-    namespace: ocp
-    name: "4.12"
-    tag: upi-installer
+  from: upi-installer
```

Comment:
If I remember right, in https://github.com/openshift/release/pull/57897, we separate the step into two small steps, `ipi-conf-aws-user-min-permissions` and `aws-provision-iam-user`. It only depends on `installer` image instead of `upi-installer`, to make it be common enough, not break any existing jobs. I think that is a good idea.

Reply:
I remember we had a such discussion,  but cannot find it.
So the expected process would be:
ref: ipi-conf-aws-user-min-permissions -> generate policy, uses installer image
ref: aws-provision-iam-user -> create IAM user, uses UPI 4.14 UPI installer
is that right?

if that, it looks like we need to create a chain, like `ipi-conf-aws-user-min-permissions`, to apply it on all `-provision` chain.

Reply:
> is that right?

Right

> if that, it looks like we need to create a chain, like ipi-conf-aws-user-min-permissions,

Sure, good idea.

Reply:
If you do `from: upi-installer` you force every workflow using this step to add the `upi-installer` to their base images. This issue came up before in https://github.com/openshift/release/pull/57759#issuecomment-2417284844 then we discussed about which version to use in https://github.com/openshift/release/pull/57893#discussion_r1803445776 and https://github.com/openshift/release/pull/57897#discussion_r1804293648